src/structures/Guild.js

```diff
@@ -1422,6 +1476,38 @@ class Guild extends Base {
       .then(() => this);
   }
 
+  /**
+   * Edits the guild's membership screening form.
+   * @param {Partial<GuildMembershipScreening>} memberScreen The membership screening data for the guild
```

Comment:
could there be something that explains what `Partial<x>` is on the docs? 
![image](https://user-images.githubusercontent.com/28943913/103251159-6df67d00-496f-11eb-918a-0eef6e79bf4c.png)